src/plugins/intel_cpu/tests/functional/shared_tests_instances/snippets/mlp_seq.cpp

```diff
@@ -0,0 +1,88 @@
+// Copyright (C) 2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "snippets/mlp.hpp"
+
+#include "utils.hpp"
+
+namespace ov {
+namespace test {
+namespace snippets {
+
+namespace {
+
+std::vector<std::vector<InputShape>> inputShape_2D() {
+    auto shapes = SNIPPETS_TESTS_STATIC_SHAPES(
+        {{1, 64}},
+        {{2, 64}},
+        {{4, 64}},
+        {{8, 64}},
+        {{16, 64}},
+        {{32, 64}},
+        {{64, 64}});
+    return shapes;
+}
+
+std::vector<std::vector<InputShape>> inputShapeQuantized_2D() {
+    auto shapes = SNIPPETS_TESTS_STATIC_SHAPES(
+        {{1, 64}},
+        {{2, 64}},
+        {{4, 64}},
+        {{8, 64}},
+        {{16, 64}},
+        {{32, 64}},
+        {{64, 64}});
+    return shapes;
+}
+
+std::vector <size_t> numHiddenLayers() {
+    return {1, 2, 3, 5, 7, 9};
+}
+
+INSTANTIATE_TEST_SUITE_P(smoke_Snippets_MLP_SEQ_2D_f32,
+                         MLP,
+                         ::testing::Combine(::testing::ValuesIn(inputShape_2D()),
+                                            ::testing::ValuesIn(precision_f32(1)),
+                                            ::testing::Values(ov::element::f32),
+                                            ::testing::ValuesIn({static_cast<size_t>(1), MLP::default_thread_count}),
+                                            ::testing::Values(1),  // Subgraph
+                                            ::testing::Values(1),  // MLP
+                                            ::testing::Values(ov::test::utils::DEVICE_CPU),
+                                            ::testing::Values(CPUTestUtils::empty_plugin_config),
+                                            ::testing::Values(2),
+                                            ::testing::ValuesIn(numHiddenLayers())),
+                         MLP::getTestCaseName);
+
+INSTANTIATE_TEST_SUITE_P(smoke_Snippets_MLP_SEQ_2D_i8,
+                         MLP,
+                         ::testing::Combine(::testing::ValuesIn(inputShape_2D()),
+                                            ::testing::ValuesIn(precision_i8(1)),
```

Comment:
I think that quantized MLP should have two tests: with fp32 input precisions and int8 input precisions. And common MLP test without FQ should have only one test with fp32 input precisions

Reply:
Applied